librz/include/rz_il.h

```diff
@@ -0,0 +1,8 @@
+#ifndef BUILD_RZ_IL_H
```

Comment:
Why `BUILD`?

Reply:
It's auto generated by my editor, I will change the setting.